config/locales/en.yml

```diff
@@ -1,33 +1,15 @@
-# Files in the config/locales directory are used for internationalization
-# and are automatically loaded by Rails. If you want to use locales other
-# than English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t "hello"
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t("hello") %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# The following keys must be escaped otherwise they will not be retrieved by
-# the default I18n backend:
-#
-# true, false, on, off, yes, no
-#
-# Instead, surround them with single quotes.
-#
-# en:
-#   "true": "foo"
-#
-# To learn more, please read the Rails Internationalization guide
-# available at http://guides.rubyonrails.org/i18n.html.
-
 en:
-  hello: "Hello world"
+  users:
+    defaults:
+      username: "Username"
+      email: "Email"
+      password: "Password"
+    new:
+      title: "create your account"
+    show:
+      welcome: "welcome to stickit <%= username %>!"
```

Comment:
I believe the right syntax for this is using a `%{ }` without the angle brackets or equal sign.
Ref: http://guides.rubyonrails.org/i18n.html#passing-variables-to-translations

Reply:
You are right! Thanks. 
Fixed in sha: d886c7f